index.js

```diff
@@ -109,22 +109,25 @@ function buildAssets (opts, callback) {
             }
             rawScss.push('@import "'+opts.variables+'";');
         }
+        rawScss.push('@import "'+getModulePath('slate')+'/source/stylesheets/_variables.scss";');
         rawScss.push('@function font-url($url){ @return url($url) }');
-        rawScss.push('@media screen { @import "'+getModulePath('slate')+'/source/stylesheets/screen.css.scss"; }');
-        rawScss.push('@media screen { .highlight._{ @import "'+getModulePath('highlight.js')+'/../styles/solarized-light"; } }');
```

Comment:
but you only import one, the second one is a forced style (just wasn't yet configureable) to have code blocks in the center area .highlight._{}